assets/airports/lszh.json

```diff
@@ -0,0 +1,1388 @@
+{
+    "radio": {
+        "twr": "Zurich Tower",
+        "app": "Zurich Arrival",
+        "dep": "Zurich Departure"
+    },
+    "icao": "LSZH",
+    "iata": "ZRH",
+    "magnetic_north": 2,
+    "ctr_radius": 80,
+    "ctr_ceiling": 19000,
+    "initial_alt": 5000,
+    "position": ["N47d27m52.92", "E8d32m57.01", "1416.0ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N47d27m52.92", "E8d32m57.01"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 200,
+        "speed": 6
+    },
+    "arrivalRunway": "14",
+    "departureRunway": "28",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 190,
+            "airspace_class": "B",
+            "poly": [
+                ["N48.14854472", "E8.212656111"],
+                ["N48.17000000", "E8.321388889"],
+                ["N48.16888889", "E8.680555556"],
+                ["N48.16666667", "E8.882777778"],
+                ["N48.16666667", "E8.953427778"],
+                ["N48.16666667", "E9.550000000"],
+                ["N47.97333333", "E9.550000000"],
+                ["N47.89000000", "E9.550000000"],
+                ["N47.89805556", "E9.835555556"],
+                ["N47.81527778", "E9.905000000"],
+                ["N47.81805556", "E9.972500000"],
+                ["N47.65944444", "E9.965833333"],
+                ["N47.53643944", "E9.948637222"],
+                ["N47.33666667", "E9.924722222"],
+                ["N47.40845194", "E9.651910556"],
+                ["N47.39611111", "E9.663333333"],
+                ["N47.39611111", "E9.281111111"],
+                ["N47.37111111", "E9.178888889"],
+                ["N47.34694444", "E9.081111111"],
+                ["N47.27333333", "E9.045555556"],
+                ["N47.27108667", "E9.036465556"],
+                ["N47.25527778", "E8.972500000"],
+                ["N47.19527778", "E8.894722222"],
+                ["N47.17630278", "E8.743665000"],
+                ["N47.14472222", "E8.492222222"],
+                ["N47.14276111", "E8.455774444"],
+                ["N47.13833333", "E8.370555556"],
+                ["N47.13138889", "E7.994722222"],
+                ["N47.14861111", "E7.982222222"],
+                ["N47.32000000", "E7.858611389"],
+                ["N47.39277778", "E7.805833333"],
+                ["N47.43055556", "E7.778611111"],
+                ["N47.56222222", "E7.763888889"],
+                ["N47.59416667", "E7.838888889"],
+                ["N47.69694444", "E8.159166667"],
+                ["N47.70416667", "E8.181388889"],
+                ["N47.73874278", "E8.183815833"],
+                ["N47.96267083", "E8.199628889"]
+            ]
+        }
+    ],
+    "fixes": {
+        "_RW10KLO02.1DME": ["N47.454888", "E8.596844"],
+        "_KLO085050": ["N47.464116","E8.667911"],

```

Comment:
Using this fix as an example but also applies to all other invisible fixes defined as a radial from a fix:
These fixes seems to be *slightly* off. The discrepancy might be due to magnetic deviation or maybe the radial just isn't precise enough. I suggest getting the actual tracks using [GPS Visualizer](http://www.gpsvisualizer.com/calculators#coordinates) For this particular example, measure the actual heading from `KLO` to `ZH502` and use that to calculate the position of `KLO085005`

Reply:
Ive only found this sid, where that applies.

Reply:
Take this image as an example:
![deges](https://user-images.githubusercontent.com/13125311/34265962-8337ac04-e66f-11e7-87eb-3758f130453a.PNG)
As you can see from the chart, the points at which aircraft intercept the 085 radial from KLO are perfectly lined up with ZH502 and KOLUL. If you take a look at the position of these fixes on the sim, they are slightly north of where they should be (caused by a discrepancy in the angle). It's a small difference but still noticeable. This also applies to many of the invisible fixes used in the other departures